project_template/_base/project_name/apps/common/utils/redis.py

```diff
@@ -0,0 +1,30 @@
+import functools
+
+from django.conf import settings
+
+import redis
+
+
+@functools.lru_cache(maxsize=128)
+def redis_client(redis_connection_url=settings.REDIS_URL):  # pragma: no cover
+    """
+    Redis client wrapped into LRU cache.
+
+    Example:
+    >>> redis_client().set('foo', 'bar')
+    >>> redis_client().get('foo')
+    """
+    return redis.StrictRedis.from_url(redis_connection_url)
+
+
+def redis_key(prefix: str, *args, delimiter=':') -> str:
```

Comment:
I can't get what issue this function has to resolve... Usually if you put something to redis in one place, you have to get it in some other place, and in this case good to have template for this key, do not misstype, and have ability to change key format easily... 
```
RESET_PASSWORD_KEY_TEMPLATE = 'user:{user_id}:comments'

def store_comments(user_id, comments):
    token = str(uuid4())
    redis.set(RESET_PASSWORD_KEY_TEMPLATE.format(user_id), comments)
  

def get_comments(user_id):
     comments = redis.get(RESET_PASSWORD_KEY_TEMPLATE.format(user_id))
```